app/views/works/_work_form_tags.html.erb

```diff
@@ -1,9 +1,18 @@
 <fieldset class="work meta">
   <legend><%= ts('Tags') %></legend>
   <h3 class="landmark heading"><%= ts('Tags') %></h3>
+
   <p class="note"><%= ts('Tags are comma separated, %{max} characters per tag.', :max => ArchiveConfig.TAG_MAX) %></p>
 
   <dl>
+    <% import ||= false
```

Comment:
So the good news is, this is what I was going to suggest when I saw the earliest version of moving the "override tags" option! (When I saw it, it wasn't part of the dl, but above the note.) The bad news is now that it covers the notes as well as the tags, I think it's better to keep it where it was. /o\ If you're adding notes but not tags, there's a very good chance you're just going to ignore the Tags section of the form completely and thus miss that it also contains an option that affects notes. At least the top section of the form is kind of a general preferences area with things you have to fill in no matter what, so there's a _slightly_ better chance you might see it (especially if you've used the form before).